main.h

```diff
@@ -22,5 +22,6 @@ int handle_char(va_list args);
 int handle_integer(va_list args);
 int handle_string(va_list args);
 int handle_format_specifier(const char *fmt, size_t *index, va_list args);
+void initialise_formatters(Formatter *specification);
```

Comment:
@tingru0515 : I think `print_by_specification` isn't in `main.h`? Adding that would be a good idea.